dev-lua/lua-openssl/lua-openssl-0.4.1.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit toolchain-funcs
+
+DESCRIPTION="OpenSSL binding for Lua"
+HOMEPAGE="https://github.com/zhaozg/lua-openssl"
+SRC_URI="https://github.com/zhaozg/lua-openssl/archive/${PV}.tar.gz -> lua-openssl-${PV}.tar.gz"
+
+LICENSE="MIT"
```

Comment:
according to https://github.com/zhaozg/lua-openssl/blob/master/LICENSE it also needs `openssl` in the license field